mne/io/kit/kit.py

```diff
@@ -504,7 +504,7 @@ def _set_dig_kit(mrk, elp, hsp):
         hsp = _read_dig_points(hsp)
     n_pts = len(hsp)
     if n_pts > KIT.DIG_POINTS:
-        hsp = _decimate_points(hsp, res=5)
+        hsp = _decimate_points(hsp, res=0.005)
```

Comment:
related, @christianbrodbeck, should we update the default value for _decimate_points and say that it should be based on `m` or we could say in general, the appropriate scale of your data